[ADD] Appraisals: Appraisal Analysis reporting (#11037)
odoo/documentation: content/applications/hr/appraisals/appraisal_analysis.rst

the search bar at the top.

.. image:: appraisal_analysis/analysis.png
   :align: center
   :alt: A report showing all the appraisals for the Appraisal Analysis report.

.. _appraisals/group-status:

Group by status
===============

When a company has a large number of employees, the default :guilabel:`Appraisal Analysis` report
may display too much information to view easily. In this scenario, viewing the data by status can be
beneficial.

First, remove the default :icon:`oi-group` :guilabel:`Department` grouping. Next, click the
:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the far-right of the search bar. Click
:guilabel:`Status` in the :icon:`oi-group` :guilabel:`Group By` column. Click off of the drop-down
menu to close it.

All the appraisals are now organized by status, in the following order: :guilabel:`Cancelled`,
:guilabel:`Done`, :guilabel:`To Start`, and :guilabel:`Appraisal Sent`.

This view makes it easy to see which appraisals need to be completed, and when, as well as which
appraisals still need to be confirmed.

.. image:: appraisal_analysis/by-status.png
   :align: center
   :alt: A report showing all the appraisals, grouped by status.

> Use case: view only your appraisals

Author: THANK you- I honestly couldn't think about how to write this without 'your'. This was tough - and now it seems obvious =D

===================================

When viewing the :guilabel:`Appraisal Analysis` report, it can be time-saving to only view the
appraisals that the signed-in user is responsible for, and hide the rest.

To view only this data, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the
far-right of the search bar, revealing a drop-down menu. Click :guilabel:`Add Custom Filter` at the
bottom of the :icon:`fa-filter` :guilabel:`Filters` column, and a :guilabel:`Add Custom Filter`
pop-up window appears.

Click into the first field, and a popover appears with a variety of options. Click the
:icon:`fa-chevron-right` :guilabel:`(right arrow)` icon after the word :guilabel:`Employee`, and

> scroll down and click on :guilabel:`Manager`. Click away to close the popover. Next, set the middle

Reviewer: ```suggestion
scroll down and click on :guilabel:`Manager`. Next, set the middle
```
Once 'manager' is selected, the pop-over goes away on its own

> field to :guilabel:`=` :guilabel:`(equal)`. Last, click into the third field and select the desired

Reviewer: ```suggestion
field to :guilabel:`=` :guilabel:`(equal)`. Last, click the third field, and select the desired
```

user from the list. When all the fields are set, click :guilabel:`Add`.

.. image:: appraisal_analysis/custom.png
   :align: center
   :alt: A customized filter to show only the user's employees.

> Now, the only appraisals that appear are the appraisals that the selected user is responsible for,

Reviewer: ```suggestion
Now, the only appraisals that appear are the appraisals that selected user is responsible for,
```

==================
Appraisal analysis
==================

The **Appraisals** app has the ability to report on all the appraisals in the system, including

> past, present, and future appraisals, and their respective statuses. This report can assist managers
> to see when the various appraisals are scheduled, and see if any appraisals are overdue or have not
> been confirmed yet.

Reviewer 2: ```suggestion
past, present, and future appraisals, and their respective statuses. This report helps managers track scheduled appraisals and identify any overdue or unconfirmed ones.
```

--> Appraisal Analysis`.

On the :guilabel:`Appraisal Analysis` page, a report loads, displaying all the appraisals in the
database. Each entry is highlighted in different color to represent their status:

+-----------+----------------------------+-----------------------------------------------+
| Color     | Status                     | Meaning                                       |
+===========+============================+===============================================+
| Yellow    |:guilabel:`Done`            | The appraisal was completed.                  |
+-----------+----------------------------+-----------------------------------------------+
| Orange    |:guilabel:`Appraisal Sent`  |The appraisal was confirmed but not completed. |
+-----------+----------------------------+-----------------------------------------------+
| Red       |:guilabel:`Cancelled`       |The appraisal was cancelled.                   |
+-----------+----------------------------+-----------------------------------------------+
| Gray      |:guilabel:`To Start`        | The appraisal was scheduled but not confirmed.|

> +-----------+----------------------------+-----------------------------------------------+

Reviewer 2: This table is a really good way to organize the info (:

Author: I started with my usual bullet list but then I realized this would be more helpful, visually =)

.. note::
   Appraisals are scheduled automatically (appear in gray), according to their respective
   :ref:`appraisals/appraisal-plan`.

The report displays the current year, in a default Gantt view, grouped by department, with the
current month highlighted.

To change the period of time that is presented by default, adjust the date settings in the top-left
of the report. The options to display are :guilabel:`Day`, :guilabel:`Week`, :guilabel:`Month`, and
:guilabel:`Year`. Use the arrows to move forward or backward in time.

At any point, click the :guilabel:`Today` button to have the Gantt view include today's date in the
view.

> To view the details of nay appraisal, click on any appraisal. A pop-over appears, displaying the

Reviewer 2: ```suggestion
To view the details of any appraisal, click on any appraisal. A pop-over appears, displaying the
```

beneficial.

First, remove the default :icon:`oi-group` :guilabel:`Department` grouping. Next, click the
:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the far-right of the search bar. Click
:guilabel:`Status` in the :icon:`oi-group` :guilabel:`Group By` column. Click off of the drop-down
menu to close it.

All the appraisals are now organized by status, in the following order: :guilabel:`Cancelled`,
:guilabel:`Done`, :guilabel:`To Start`, and :guilabel:`Appraisal Sent`.

This view makes it easy to see which appraisals need to be completed, and when, as well as which
appraisals still need to be confirmed.

.. image:: appraisal_analysis/by-status.png
   :align: center
   :alt: A report showing all the appraisals, grouped by status.

Use case: view only your appraisals
===================================

When viewing the :guilabel:`Appraisal Analysis` report, it can be time-saving to only view the
appraisals that the signed-in user is responsible for, and hide the rest.

To view only this data, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the
far-right of the search bar, revealing a drop-down menu. Click :guilabel:`Add Custom Filter` at the

> bottom of the :icon:`fa-filter` :guilabel:`Filters` column, and a :guilabel:`Add Cusomt Filter`

Reviewer 2: ```suggestion
bottom of the :icon:`fa-filter` :guilabel:`Filters` column, and a :guilabel:`Add Custom Filter`
```

pop-up window appears.

> Click into the first field, and a popover appears with a variety of options. Click the

Reviewer 2: Just wanted to comment that this custom filter is so amazing. I didn't know about the right chevron arrow. How did you figure out this filter?! :open_mouth: :exploding_head: 
You've done amazing and detailed research for this :pleading_face:

Author: It was all trial and error! I was like "I want to show X- HOW DO I DO IT?!". So yeah, it was a LOT of frustrating trial and error to figure it out! =D

:icon:`fa-chevron-right` :guilabel:`(right arrow)` icon after the word :guilabel:`Employee`, and

> scroll down and click on :guilabel:`Manager`. Click away to close the popover. Change the middle
> drop-down to :guilabel:`=` :guilabel:`(equal)`. Last, click into the third drop-down and select

Reviewer 2: ```suggestion
scroll down and click on :guilabel:`Manager`. Click away to close the popover. Next, set the middle
field to :guilabel:`=` :guilabel:`(equal)`. Last, click into the third field and select
```

the desired user from the list. When all the fields are set, click :guilabel:`Add`.

> .. image:: appraisal_analysis/custom.png

Reviewer 2: this image a little stretched out, vertically. Let's try uploading this image again (:

![image](https://github.com/user-attachments/assets/c895c5ab-b4f6-4602-a458-c371248ac97a)

Author: Agree- I took this photo over 5 times, and it kept doing something wonky so I thought, I'll redo this tomorrow but first,  let's get this doc reviewed  =D

Author: OK, I FINALLY got it to work! I had to take a full-sized image, then change the dimensions in ANOTHER app since it kept getting messed up with Nimbus... OMG that took forever =D

   :align: center

>    :alt: A cusotmized filter to show only the user's employees.

Reviewer 2: ```suggestion
   :alt: A customized filter to show only the user's employees.
```
